src/main/controllers/DailyCauseListController.ts

```diff
@@ -0,0 +1,32 @@
+import {Response} from 'express';
+import {PipRequest} from '../models/request/PipRequest';
+import {cloneDeep} from 'lodash';
+import moment from 'moment';
+import { SummaryOfPublicationsService } from '../service/summaryOfPublicationsService';
+
+const publicationService = new SummaryOfPublicationsService();
```

Comment:
youre in DailyCauseList, so either the SummaryOfPublicationService, can be made generic and moved to publicationService (recommended) 
or you need to have a service specific to DailyCauseList that repeats the functionalty in SummaryOfPublicationService (not recommended)

Reply:
Agree. Change the code to use publicationService.